### PR TITLE
Update SAML JavaDoc to reference specification

### PR DESCRIPTION
**Pull request by marcusdacoregio**
Closes gh-9510

### CODE REVIEW

**Comment by marcusdacoregio** [comment]
I went through the docs that are still referencing the **Shibboleth** wiki but they are still valid references to their implementation

File: saml2/saml2-service-provider/core/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public String getRegistrationId() {
 
 	/**
 	 * Get the relying party's <a href=
-	 * "https://wiki.shibboleth.net/confluence/display/CONCEPT/EntityNaming">EntityID</a>.
+	 * "https://www.oasis-open.org/committees/download.php/51890/SAML%20MD%20simplified%20overview.pdf#2.9%20EntityDescriptor">EntityID</a>.
```
Comment (jzheaux) [inline saml2/saml2-service-provider/core/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java:137]: Will you please update the copyright header in this file to include the current year?

**Review by jzheaux — CHANGES_REQUESTED** [review]
Thanks for the PR, @marcusdacoregio! Please see my feedback inline.

File: saml2/saml2-service-provider/core/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java (hunk 1 above)
Comment (marcusdacoregio) [reply saml2/saml2-service-provider/core/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java:137]: Sure, thanks for pointing this out

**Comment by jzheaux** [comment]
Thanks, @marcusdacoregio! This is now merged into `main`.

**Pull request merged**